spec/spec_helper.rb

```diff
@@ -113,7 +117,7 @@ def image_path(path, options = {})
 end
 
 VCR.configure do |c|
-  c.cassette_library_dir = Rails.root.join('spec/vcr_cassettes')
+  c.cassette_library_dir = File.join(external_tower_gem_dir, 'spec/vcr_cassettes')
```

Comment:
I like my solution better... https://github.com/ManageIQ/manageiq/pull/15594/files#diff-88d3c9989ff613ced10a9fe635834148R2
This change breaks everything else that uses VCR in ManageIQ